### DISCUSSION TITLE
Lost configuration after docker pull update

### DISCUSSION THREAD

**Discussion by Bertels999** [zwave-js/zwave-js-ui#3470] [Q&A]
Whenever I update ZWaveJSUI (installed using docker on my raspberry pi), I use the method described here: https://github.com/zwave-js/zwave-js-ui/discussions/2748.

Basically, it boils down to : docker-compose pull  && docker-compose up -d

The big disadvantage of this approach is that I must configure ZWaveJSUI from scratch and that it starts scanning all the devices again rebuilding the nodes. Is there an easier way, so that, at least, the existing configuration is reused so that I don't have to reconfigure ZWaveJSUI again?

It would also be useful that the documentation point out how to update to a new version. This topic is currently lacking.

Regards Bert

**Comment by robertsLando** [comment]
> The big disadvantage of this approach is that I must configure ZWaveJSUI from scratch and that it starts scanning all the devices again rebuilding the nodes. Is there an easier way, so that, at least, the existing configuration is reused so that I don't have to reconfigure ZWaveJSUI again?

This just measn you didn't configured volumes correctly on your docker-compose file. Could you paste your docker-compose file here please?

**Reply by Bertels999** [reply]
Ah, thanks. I corrected my docker-compose file. Regards Bert

**Reply by robertsLando** [reply]
If you show me I could verify if you did it correctly to prevent the above error

**Comment by Bertels999** [comment]
Sure and thank you!
[docker-compose.txt](https://github.com/zwave-js/zwave-js-ui/files/13601950/docker-compose.txt)

changed the extension to txt otherwise it would not upload.

**Reply by robertsLando** [reply]
That's ok now, it will create a  `store` directory next to the docker compose file on boot and everything will be stored there
